ContentBuildingReader.cpp

```diff
@@ -418,11 +416,10 @@ bool addSingleBuildingConfig( TiXmlElement* elemRoot,  std::vector<std::unique_p
     }
     else if (strGameCustom && strGameCustom[0])
     {
-        for (size_t i = 0; i < df::global::world->raws.buildings.all.size(); i++)
-        {
-            if (strcmp(strGameCustom, df::global::world->raws.buildings.all[i]->code.c_str()) == 0)
-                custom = i;
-        }
+        auto bld = df::global::world->raws.buildings.all;
```

Comment:
will this become `auto & bld` automatically?

Reply:
that does need to be a reference, thank you